MobiFlight/Joysticks/JoystickManager.cs

```diff
@@ -154,8 +154,7 @@ public void Connect(IntPtr Handle)
                 Log.Instance.log($"Adding attached joystick device: {d.InstanceName} Buttons: {js.Capabilities.ButtonCount} Axis: {js.Capabilities.AxeCount}.", LogSeverity.Info);
                 js.Connect(Handle);
                 joysticks.Add(js);
-                js.OnButtonPressed += Js_OnButtonPressed;
-                js.OnAxisChanged += Js_OnAxisChanged;
```

Comment:
Why did OnAxisChanged get removed?

Reply:
If I was not misled, it is dead uncalled code.